vendor-bundles.webpack.config.js

```diff
@@ -1,76 +1,63 @@
-"use strict";
-/* eslint-disable */
+const webpack = require("webpack");
+const path = require("path");
 
-let webpack = require('webpack');
-const path = require('path');
+// we don't need sourcemaps on circleci
+const sourceMap = process.env.IS_CIRCLECI ? "" : "source-map";
```

Comment:
@alisman minor stylistic thing but I tend to prefer `DISABLE_SOURCEMAP=true` and then set `DISABLE_SOURCEMAP=true` on CircleCI. You might want to reuse this configuration elsewhere that's not CircleCI.

Reply:
personal preference thing though, i can see advantages of doing it the other way around